dev-python/rtslib-fb/rtslib-fb-2.1.75.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
EAPI 8 and PEP517 build, please.

Reply:
unsure what is required to verify this change, please clarify